src/Controls/src/BindingSourceGen/BindingSourceGenerator.cs

```diff
@@ -142,22 +138,31 @@ static bool ParsePathNullableEnabled(CSharpSyntaxNode? expressionSyntax, Generat
 			{
 				return false;
 			}; // TODO
-			parts.Add(new PathPart(member, isNodeNullable, index));
+			if (!enabledNullable && typeInfo.IsReferenceType)
```

Comment:
If it's not a reference type, it could still be `Nullable<T>` and then it is a nullable value type